### PR TITLE
bump bugfix: Hide zero value amount for wallet connect and fix typo

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [CSI-4185 Hide zero value amount for wallet connect and fix typo](https://app.clickup.com/t/9002019994/CSI-4185)

File: cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Summary.tsx [hunk 1]
```diff
@@ -129,6 +130,7 @@ const Summary: React.FC<StepComponentProps> = ({
   const { addCustomAccountSyncItemFromCoin } = useSync();
 
   const handleSend = async () => {
+    return;
```
Comment (irshadCypherock) [inline cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Summary.tsx]: Maybe this was added to commit by mistake. If so, please remove it.

File: cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Summary.tsx [hunk 2]
```diff
@@ -277,27 +279,29 @@ const Summary: React.FC<StepComponentProps> = ({
...
-                    2,
-                    true
-                  )}) `
-            }
-            verified
-          />
-        )}
+        {activeButton === 0 &&
+          (triggeredBy !== TriggeredBy.WalletConnect ||
+            parseInt(batchRecipientData[0].amount, 10) !== 0) && (
```
Comment (irshadCypherock) [inline cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Summary.tsx:283]: The condition here and in `cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Recipient.tsx` Line `843` does not match.

File: cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Verify.tsx [hunk 3]
```diff
@@ -170,24 +171,31 @@ const Verify = (props: any) => {
...
-                      parseFloat(sendTransaction.sendMaxAmount) *
-                        parseFloat(coinPrice),
-                      2,
-                      true
-                    )}) `
-              }
-              verified={sendTransaction.verified}
-            />
+            {(triggeredBy === TriggeredBy.WalletConnect &&
+              parseInt(batchRecipientData[0].amount, 10) === 0) || (
```
Comment (irshadCypherock) [inline cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Verify.tsx]: Same here.

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: cysync/release_notes.json [hunk 4]
```diff
@@ -1,3 +1,3 @@
 {
-  "notes": "# v1.2.0\n\n### 🚀 Features\n\n- Token Support for EVM chains\n\n### 🐛 Bugfixes\n\n- Fixed latest price update\n\n# v1.1.786\n\n### 🚀 Features\n\n- Support for Arbitrum One and Optimism\n- WalletConnect support\n\n# v1.1.282\n\n### 🐛 Bugfixes\n\n- Bugfixes for Bitcoin legacy\n\n# v1.1.281\n\n### 🚀 Features\n\n- Multi account support is here\n\n### 🌟 Improvements\n\n- Improved auto update\n"
+  "notes": "# v1.2.0\r\n\r\n### 🚀 Features\r\n\r\n- Token Support for EVM chains\r\n\r\n### 🐛 Bugfixes\r\n\r\n- Fixed latest price update\r\n\r\n# v1.1.786\r\n\r\n### 🚀 Features\r\n\r\n- Support for Arbitrum One and Optimism\r\n- WalletConnect support\r\n\r\n# v1.1.282\r\n\r\n### 🐛 Bugfixes\r\n\r\n- Bugfixes for Bitcoin legacy\r\n\r\n# v1.1.281\r\n\r\n### 🚀 Features\r\n\r\n- Multi account support is here\r\n\r\n### 🌟 Improvements\r\n\r\n- Improved auto update\r\n"
```
Comment (irshadCypherock) [inline cysync/release_notes.json]: Can you check why this was changed by the precommit?

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Summary.tsx (hunk 2 above)
Comment (ghost) [reply cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Summary.tsx:283]: the recipient file can directly use the value provided by wallet connect but after that, we have to rely on on batchRecipientData[0] for value

Comment (irshadCypherock) [reply cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Summary.tsx:283]: If the first condition is satisfied, I think we can assume the amount was never changed. 

Can you add a boolean value in `send/stepperForm.txn` which will tell us if the amount field should be displayed or not?

Comment (ghost) [reply cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Summary.tsx:283]: I just added this same condition in all three files

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**